src/docs/deployment/cd.md

```diff
@@ -7,11 +7,33 @@ Follow continuous delivery best practices with Flutter to make sure your
 application is delivered to your beta testers and validated on a frequent basis
 without resorting to manual workflows.
 
+
+## CI/CD Options
+
+There are a number of continuous integration (CI) and continuous delivery (CD)
+options available to help automate the delivery of your application.
+
+### All-in-one type of options, Flutter functionality built in XXX
+* [Codemagic][]
+* [Bitrise][]
+* [Appcircle][]
+
+### General CI options XXX
+* [Cirrus][]
+    * Example: Flutter framework repository's [Cirrus script][].
+* [GitHub Actions][]
+    * Example: Flutter Gallery's [Github Actions workflows][]
+    * Example: [Github Action in Flutter Project][]
+* [Travis][]
+* [GitLab][]
+
+This guide shows how to set up fastlane and then integrate it with 
+your existing testing and continuous integration (CI) workflows ("General CI options" above XXX).
```

Comment:
i.e. you likely wouldn't use fastlane unless you were using Cirrus, GitHub Actions, Travis, or Gitlab